README.md

```diff
@@ -2,32 +2,110 @@
 
 [![Latest Version on Packagist](https://img.shields.io/packagist/v/cybex/laravel-protector.svg?style=flat-square)](https://packagist.org/packages/cybex/laravel-protector)
 
-Protect Databases by generating Backups and Import these on non-productive Environments.
-Authenticate users with the usage of Laravel Sanctum Tokens.
-Encrypt your database dumps with Sodium Encryption.
+Allows a client to download and import a live server database backup.
+
+Common usage scenarios
+- Non-productive developer machines download the live server database. 
+- A central backup server collects backups from multiple live servers.
+
+Feature set:
+ 
+- Download and Import databases from a server.
+- Export the local database to a file.
+- User authentication through Laravel Sanctum Tokens.
+- Transport encryption using Sodium.
 
 ## Installation
 
-You can install the package via composer:
+You can install the package via composer, and run its database migration:
 
 ```bash
 composer require cybex/laravel-protector
+artisan migrate
+```
+
+In your User model class, add the following Trait, then commit and publish the change.
+```php
+class User extends Authenticatable
+{
+    use HasApiTokens;
+```
+
+## Setup
+
+This package would usually run on both a server and a client machine of the same software repository.
+You set up authorized developers on the server, and give them the key for their local machine.
+
+### On the Client
+
+As a developer, run the following command
+```bash
+php artisan protector:keys
+```
+
+You will receive
+- the public key to give to your server admin
+- the private key to save in your .env file
+
+Your server admin will then give you the token and server-url to save in your .env file.
+Unless specified otherwise in your software, the .env keys are
+
+```bash
+PROTECTOR_AUTH_TOKEN=
+PROTECTOR_SERVER_URL=
+```
```

Comment:
Might want to add `PROTECTOR_PRIVATE_KEY` for the private key.

Reply:
Not too necessary tho since it is already stated when using the `protector:keys` command.

Reply:
yes, this is currently regarded as a second, separate step